applications/luci-app-alist/htdocs/luci-static/resources/view/alist/config.js

```diff
@@ -0,0 +1,151 @@
+			window.location.hostname, port, _('Open Web Interface'));
+		renderHTML = spanTemp.format('green', _('AList'), _('RUNNING')) + button;
+	} else {
+		renderHTML = spanTemp.format('red', _('AList'), _('NOT RUNNING'));
+	}
+
+	return renderHTML;
+}
+
+var stubValidator = {
+	factory: validation,
+	apply: function(type, value, args) {
+		if (value != null)
+			this.value = value;
+
+		return validation.types[type].apply(this, args);
+	},
+	assert: function(condition) {
+		return !!condition;
+	}
+};
+
+return view.extend({
+	load: function() {
+		return Promise.all([
+			uci.load('alist')
+		]);
+	},
+
+	render: function(data) {
+		var m, s, o;
+		var webport = uci.get(data[0], 'config', 'listen_http_port') || '5244';
+
+		m = new form.Map('alist', _('AList'),
+			_('A file list/WebDAV program that supports multiple storages, powered by Gin and Solidjs.<br />' +
+				'Default login username is <code>admin</code> and password is <code>password</code>.'));
+
+		s = m.section(form.TypedSection);
+		s.anonymous = true;
+		s.render = function () {
+			poll.add(function () {
+				return L.resolveDefault(getServiceStatus()).then(function (res) {
+					var view = document.getElementById('service_status');
+					view.innerHTML = renderStatus(res, webport);
+				});
+			});
+
+			return E('div', { class: 'cbi-section', id: 'status_bar' }, [
+					E('p', { id: 'service_status' }, _('Collecting data...'))
+			]);
+		}
+
+		s = m.section(form.NamedSection, 'config', 'alist');
+
+		o = s.option(form.Flag, 'enabled', _('Enable'));
+		o.default = o.disabled;
+		o.rmempty = false;
+
+		o = s.option(form.Value, 'listen_addr', _('Listen address'));
+		o.default = '0.0.0.0';
+		o.rmempty = false;
+		o.validate = function(section_id, value) {
+			if (!value)
+				return _('Expecting: %s').format(_('non-empty value'));
+
+			var m4 = value.match(/^([^\[\]:]+)$/),
+			    m6 = value.match(/^\[(.+)\]$/ );
+
+			if ((!m4 && !m6) || !stubValidator.apply('ipaddr', m4 ? m4[1] : m6[1]))
+				return _('Expecting: %s').format(_('valid IP address'));
+
+			return true;
+		}
+
+		o = s.option(form.Value, 'listen_http_port', _('Listen port'));
+		o.datatype = 'port';
+		o.default = '5244';
+		o.rmempty = false;
+
+		o = s.option(form.Value, 'site_login_expire', _('Login expiration time'),
+			_('User login expiration time (in hours).'));
+		o.datatype = 'uinteger';
+		o.default = '48';
+		o.rmempty = false;
+
+		o = s.option(form.Value, 'site_max_connections', _('Max connections'),
+			_('The maximum number of concurrent connections at the same time (0 = unlimited).'));
+		o.datatype = 'uinteger';
+		o.default = '0';
+		o.rmempty = false;
+
+		o = s.option(form.Flag, 'site_tls_insecure', _('Allow insecure connection'),
+			_('Allow connection even if the remote TLS certificate is invalid (<strong>not recommended</strong>).'));
+		o.default = o.disabled;
```

Comment:
`o.default = o.disabled;`

looks like the line should be removed

Reply:
probably.. but looks like it's still common to declear this explicitly

Reply:
The init script you submitted does not have a default. So it would be best to remove the default here.

Reply:
I mean you really take a value of the `o.disabled` which is always probably false or undefined.
The `default` for flag values is false

Reply:
> The init script you submitted does not have a default. So it would be best to remove the default here.

In my function `uci_json_add_boolean` it defaults to `false` ;)

> The `default` for flag values is false

Yes I see it so I said "it's still common to declear this explicitly". Anyway removed ;)